test/test_transforms.py

```diff
@@ -1308,13 +1311,13 @@ def test_reward_scaling(self, batch, scale, loc, keys, device):
         td_copy = td.clone()
         reward_scaling(td)
         for key in keys_total:
-            assert (td.get(key) == td_copy.get(key).mul_(scale).add_(loc)).all()
+            if standard_normal:
+                assert (td.get(key) == ((td_copy.get(key) - loc) / scale)).all()
```

Comment:
maybe `torch.testing.assert_close(a, b)` instead? to avoid numerical precision errors

Reply:
Got it!